stuffer/s2n_stuffer_network_order.c

```diff
@@ -154,7 +152,7 @@ int s2n_stuffer_read_uint64(struct s2n_stuffer *stuffer, uint64_t * u)
 
 int s2n_stuffer_write_uint64(struct s2n_stuffer *stuffer, const uint64_t u)
 {
-    GUARD(s2n_stuffer_write_network_order(stuffer, u >> (sizeof(uint32_t) * 8), sizeof(uint32_t)));
+    GUARD(s2n_stuffer_write_network_order(stuffer, u >> (sizeof(uint32_t) * sizeof(u)), sizeof(uint32_t)));
```

Comment:
I think that `8` is actually `CHAR_BIT`.

Reply:
It might be nice documentation to add a `#define SIZEOF_IN_BITS( t ) (sizeof(t) * CHAR_BIT)` macro, and use it.

Reply:
done.